Maintainers_Guide.md

```diff
@@ -0,0 +1,62 @@
+# BIDS Maintainers Group guide
+
+The BIDS Maintainers are a key group of members that ensure the BIDS infrastructure is kept up.
+This group has wide-ranging responsibilities that enables BIDS to continue to grow and succeed.
+
+## Responsibilities
+
+The primary responsible of a maintainer is ensuring the infrastructure that the BIDS community uses is handled and kept up to date.
+Here we list several examples of common responsibilities for a BIDS maintainer.
+Please note that one maintainer need not do all of these tasks.
+They may be spread across the team according to the initiative of each maintainer and/or the direction of the lead maintainer.
+
+* Triaging issues and PRs then notifying appropriate reviewers in a timely manner
+* Facilitating discussions
+* Ensuring the Contributors list is up to date and assisting contributors with adding their credits
+     * Perform community inquiries to ensure contributors are credited in the [Contributors appendix](https://bids-specification.readthedocs.io/en/stable/99-appendices/01-contributors.html)
+     * Verify contributing members (e.g., author of a pull request) are also identified in the [Contributors appendix](https://bids-specification.readthedocs.io/en/stable/99-appendices/01-contributors.html)
+     * Ultimately, what constitutes a contribution is left to the discretion of the maintainers
+* Assisting with voting and elections
+* Proposing and implementing enhancements to the infrastructure
+
+Maintainers service time will be agreed upon between the lead maintainer and the interested maintainer.
+Maintainers will be reviewed every 3 - 6 months.
+This will be determined on a case by case basis at the discretion of the lead maintainer.
+There is no limit to the number of appointments a member may obtain.
+
+## Admission process
+
+### Eligibility
+
+To become eligible to join the Maintainers group the prospective member would have been contributing to BIDS for at least 3 months and have a good overview of the content and infrastructure surrounding BIDS.
+Contributing to BIDS can be in the form of submitting pull requests, commenting on specification or project documents, or actively participating on a BIDS extension proposal (BEP).
```

Comment:
Perhaps we can add that developing software USING BIDS is also fine. ... these people would also have a deep knowledge of BIDS (to be assessed by lead maintainer and rest of maintainers)